### PR TITLE
Created `Nuspec` and `NuspecField` interfaces

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #141 
Created `Nuspec` interface and implementation, inlined some methods from the old class. 
Created `NuspecField` interfaces and made `Version` implement it.

Next step will be somehow correct `PackageId` (and extract class to generate keys) to implement `NuspecField` too.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@genryxy could you please take a look?

File: src/main/java/com/artipie/nuget/metadata/NuspecField.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * Nuspec xml metadata field.
+ * @since 0.6
+ */
+public interface NuspecField {
+
+    /**
+     * Original row value (as it was in xml).
+     * @return String value
+     */
+    String row();
```
Comment (genryxy) [inline src/main/java/com/artipie/nuget/metadata/NuspecField.java:17]: @olenagerasimova here should be `raw`?

**Comment by olenagerasimova** [comment]
@genryxy sure, thanks

**Comment by genryxy** [comment]
@olenagerasimova there is a violation after renaming

**Comment by olenagerasimova** [comment]
@genryxy sorry, corrected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**